implementation/src/test/java/io/smallrye/config/ConfigMappingLoaderTest.java

```diff
@@ -26,8 +26,7 @@ void multipleLoads() {
     void loadManually() {
         List<ConfigMappingMetadata> configMappingsMetadata = ConfigMappingLoader.getConfigMappingsMetadata(ServerManual.class);
         configMappingsMetadata.forEach(
-                mappingMetadata -> ConfigMappingLoader.loadClass(ServerManual.class,
-                        mappingMetadata.getClassName(), mappingMetadata.getClassBytes()));
+                mappingMetadata -> ConfigMappingLoader.loadClass(ServerManual.class, mappingMetadata));
```

Comment:
This is typically a case where a lambda is totally useless and costs memory for nothing.

Reply:
Nice catch, but this is a test and will never go into production ;)

Reply:
Yes, but I guess the comment was more general, since we are abusing lambdas on the runtime code :)

I'm refactoring some pieces to remove them.

Reply:
@gastaldi sure but it was to illustrate this issue: https://github.com/quarkusio/quarkus/issues/12960#issuecomment-716733836